ephios/core/signup/methods.py

```diff
@@ -359,9 +359,13 @@ def perform_signup(self, participant: AbstractParticipant, **kwargs) -> Abstract
         Configure a participation object for the given participant according to the method's configuration.
         `kwargs` may contain further instructions from e.g. a form.
         """
+        from ephios.core.notifications.types import ResponsibleParticipationRequested
+
         if errors := self.get_signup_errors(participant):
             raise ParticipationError(errors)
-        return self.get_participation_for(participant)
+        participation = self.get_participation_for(participant)
+        ResponsibleParticipationRequested.send(participation)
+        return participation
```

Comment:
Sending this notification fails with instant signup as the participation doesn't have an id here so looking it up while sending fails. Creating the notification object probably needs to happen somewhere else.
Lets include a regression test?!

Reply:
can you explain how to reproduce this? for me notifications with instant_confirmation work flawlessly

Reply:
Showed you in the call